### PR TITLE
(Chore) : Added test to set the output level of gain Node

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

updated gain test and added test to set the output level of gain Node.

### CODE REVIEW

**Comment by Abhijay007** [comment]
refer : #462 

cc : @davepagurek

File: test/tests/p5.Gain.js [hunk 1]
```diff
@@ -36,6 +36,23 @@ describe('p5.Gain', function () {
...
+        osc.amp(1);
+        osc.start();
+        osc.disconnect();
+
+        mainGainNode.setInput(osc);
+        amplitude.setInput(mainGainNode);
+
+        mainGainNode.amp(0.5);
+        setTimeout(function () {
+          expect(amplitude.getLevel()).to.be.closeTo(0.25, 0.125);
```
Comment (davepagurek) [inline test/tests/p5.Gain.js:53]: Thanks for moving this over! Taking a closer look, I think the test suite might not wait for the `setTimeout` callback to run before thinking the test has completed. Mind using the form of test that takes a `done` callback to make sure the test only finishes once the timer finishes? Like this:
```js
it('can set the output level of gain node', (done) => {
  // [the rest of the test here]
  setTimeout(function() {
    expect(amplitude.getLevel()).to.be.closeTo(0.25, 0.125);
    done();
  }, 100);
});

Comment (Abhijay007) [reply test/tests/p5.Gain.js:53]: sure will do :)

**Comment by Abhijay007** [comment]
requested changes are done, used `done` callback to make sure the test only finishes once the timer finishes.

cc : @davepagurek

**Review by davepagurek — APPROVED** [review]
Thanks @Abhijay007, looks good!

**Pull request merged**